### ISSUE TITLE
Sound and Vibrate not work at default config

### ISSUE THREAD

**Issue by duytq94** [MaikuB/flutter_local_notifications#94]
Hi @MaikuB 
First, thanks for your plugin.
But when I run my project on a real Android device (Xiaomi note 4). Sound and Vibrate not work at all.
```
    var androidPlatformChannelSpecifics = new AndroidNotificationDetails(
      'your channel id',
      'your channel name',
      'your channel description',
      importance: Importance.Max,
      priority: Priority.High,
    );
```
Have you test this case yet.
Thanks!

**Comment by duytq94** [comment]
With Sony z3, just play sound, vibrate not work.

**Comment by MaikuB** [comment]
Yes I have, worked fine on my Pixel 2 XL and when I tried it with emulators for older Android versions. There shouldn't be any difference in logic of the Android APIs handle sound and vibration as far as i know. Perhaps a silly question but just to double check, you've actually turned on the sound for the phones you're testing with?

If the problem still persists, would you be able to debug with a local copy of the code and perhaps try to comment out the code that calls `setSound(...)` and `setVibrationPattern(...)`

**Comment by duytq94** [comment]
Yup, I turn on sound but seem these phones are weird.
I'll try another way.
Thank you.
